src/ensembl/src/shared/hooks/useAnalyticsService.ts

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
`useAnalyticsService` => `useCommonAnalytics`?

Reply:
That's done